lms/services/blackboard_api/_basic.py

```diff
@@ -38,7 +38,7 @@ class Meta:
     def parse(self, *args, **kwargs):
         try:
             return super().parse(*args, **kwargs)
-        except ValidationError:
+        except ExternalRequestError:
```

Comment:
The `RequestsResponseSchema` subclass now raises `ExternalRequestError` rather than `ValidationError`, so `BlackboardErrorResponseSchema` has to catch `ExternalRequestError` instead